### DISCUSSION TITLE
Varnish config

### DISCUSSION THREAD

**Discussion by ghost** [KiweeEu/magento2-on-kubernetes#291] [Q&A]
Should we configuration on admin console for varnish cache?

**Comment by macieklewkowicz** [comment]
Varnish configuration is in https://github.com/KiweeEu/magento2-on-kubernetes/blob/main/deploy/bases/varnish/config/default.vcl. It was based on the VCL generated from Magento admin.

Varnish is also enabled, configured, and locked in Magento config using environment variables in https://github.com/KiweeEu/magento2-on-kubernetes/blob/main/deploy/step-3/config/additional.env.

**Comment by ghost** [comment]
No need to enable full page cache in admin console?

**Comment by macieklewkowicz** [comment]
Correct, it should already be enabled and locked.
